src/stravalib/model.py

```diff
@@ -880,6 +884,7 @@ class SummaryActivity(MetaActivity, strava_model.SummaryActivity):
     sport_type: Optional[RelaxedSportType] = None
 
 
+# TODO: should this be named Detailed Activity?
```

Comment:
i suspect this will be addressed in a future pr. Activities hasn't been done yet.

Reply:
Activities were done already. I just hesitated to rename this class to DetailedActivity. But since you started doing this for clubs, I think it's a good idea to be consistent with the Strava API docs. I'll deal with that in a final cleanup PR.

Reply:
perfect!